src/sidebar/store/modules/groups.js

```diff
@@ -35,6 +34,10 @@ const initialState = {
 /** @typedef {typeof initialState} State */
 
 const reducers = {
+  /**
+   * @param {State} state
+   * @param {{ filteredGroupIds: string[] }} action
```

Comment:
For here and `FOCUS_GROUP` and another place or two in this module, it is an option to use the `Group["id"]` type instead of `string`. This is done on line 125. I kinda think it's helpful but I'm not passionate about it.

Reply:
Hmm ... I'm not sure about this. The majority of references to entity IDs (all bar 3) in the app do not follow this pattern and in the general case it would not always be appropriate (eg. you couldn't use this for `Annotation.id` because `id` is an optional field there). On the other hand it does provide some information about the expected provenance of values. I think I'm going to land this and come back to this detail later.